Buildings/package.mo

```diff
@@ -1066,6 +1066,12 @@ that can lead to wrong simulation results):
                        <a href=\"https://github.com/lbl-srg/modelica-buildings/issues/3151\">Buildings, issue 3151</a>
     </td>
 </tr>
+<tr><td valign=\"top\">Buildings.Fluid.HeatExchangers.DryCoilDiscretized
```

Comment:
@hcasperfu : Note that this change only needs to be added to the release notes on the `maint_9.1.x` branch, but not the master. The reason is that we will release 9.1.1 (or 9.2.0), and then copy the release notes of 9.1.1 to the release history in the master before releasing 10.0.0. Hence, this change is not really done new in 10.0.0, but rather in its earlier release 9.1.1.

Reply:
Understood! I will work on the maintenance branches once the tests pass here and then move the release notes.

Reply:
Reverted in 234312e.